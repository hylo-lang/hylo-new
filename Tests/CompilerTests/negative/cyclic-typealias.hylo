//! no-std

type X = Y
type Y = Z
type Z = X
