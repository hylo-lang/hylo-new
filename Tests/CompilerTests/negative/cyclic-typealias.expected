negative/cyclic-typealias.hylo:3.6-7: error: definition of 'X' refers to itself
type X = Y
     ^
